Microsoft.Toolkit.Uwp.Input.GazeInteraction/GazePointer.h

```diff
@@ -0,0 +1,176 @@
+//Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license.
+//See LICENSE in the project root for license information.
+
+#pragma once
+#pragma warning(disable:4453)
+
+#include "IGazeFilter.h"
+#include "GazeCursor.h"
+#include "PointerState.h"
+
+using namespace Platform;
+using namespace Platform::Collections;
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+using namespace Windows::Devices::Enumeration;
+using namespace Windows::Devices::HumanInterfaceDevice;
+using namespace Windows::UI::Core;
+using namespace Windows::Devices::Input::Preview;
+
+namespace Shapes = Windows::UI::Xaml::Shapes;
+
+BEGIN_NAMESPACE_GAZE_INPUT
+
+// units in microseconds
+const TimeSpan DEFAULT_FIXATION_DELAY = TimeSpanFromMicroseconds(350000);
+const TimeSpan DEFAULT_DWELL_DELAY = TimeSpanFromMicroseconds(400000);
+const TimeSpan DEFAULT_DWELL_REPEAT_DELAY = TimeSpanFromMicroseconds(400000);
+const TimeSpan DEFAULT_REPEAT_DELAY = TimeSpanFromMicroseconds(400000);
+const TimeSpan DEFAULT_THRESHOLD_DELAY = TimeSpanFromMicroseconds(50000);
+const TimeSpan DEFAULT_MAX_HISTORY_DURATION = TimeSpanFromMicroseconds(3000000);
+const TimeSpan MAX_SINGLE_SAMPLE_DURATION = TimeSpanFromMicroseconds(100000);
+
+const TimeSpan GAZE_IDLE_TIME{ 25000000 };
+
+ref class GazeTargetItem;
+ref struct GazeHistoryItem;
+
+public ref class GazePointer sealed
+{
+public:
+    virtual ~GazePointer();
+
+    void LoadSettings(ValueSet^ settings);
+
+    void Reset();
+    void SetElementStateDelay(UIElement ^element, PointerState pointerState, TimeSpan stateDelay);
+    TimeSpan GetElementStateDelay(UIElement ^element, DependencyProperty^ property, TimeSpan defaultValue);
+    TimeSpan GetElementStateDelay(UIElement^ element, PointerState pointerState);
+
+    // Provide a configurable delay for when the EyesOffDelay event is fired
+    // GOTCHA: this value requires that _eyesOffTimer is instantiated so that it
+    // can update the timer interval 
+    property TimeSpan EyesOffDelay
+    {
+        TimeSpan get() { return _eyesOffDelay; }
+        void set(TimeSpan value)
+        {
+            _eyesOffDelay = value;
+
+            // convert GAZE_IDLE_TIME units (microseconds) to 100-nanosecond units used
+            // by TimeSpan struct
+            _eyesOffTimer->Interval = EyesOffDelay;
+        }
+    }
+
+    // Pluggable filter for eye tracking sample data. This defaults to being set to the
+    // NullFilter which performs no filtering of input samples.
+    property IGazeFilter^ Filter;
+
+    property bool IsCursorVisible
+    {
+        bool get() { return _gazeCursor->IsCursorVisible; }
+        void set(bool value) { _gazeCursor->IsCursorVisible = value; }
+    }
+
+    property int CursorRadius
+    {
+        int get() { return _gazeCursor->CursorRadius; }
+        void set(int value) { _gazeCursor->CursorRadius = value; }
+    }
```

Comment:
Not sure I get what these setters do, so just checking if they should be public?